signing-and-verification-workflow.md

```diff
@@ -0,0 +1,52 @@
+# Signing and Verification Workflow
+
+This document describes how Notary v2 signs and verifies OCI artifacts.
+
+## Signing workflow
+
+The user wants to sign a locally available OCI artifact and push it to a repository(along with signature).
+
+### Prerequisite
+
+- User has access to the signing certificate and private key.
```

Comment:
I've seen concern with the term "access". The signing cert may be in a remote, secured location, the user "has access", but can't take the private certificate.

Suggestion:
- User has access to the signing certificate private key or a remote signing service through a notation plug-in.